styles/main.css

```diff
@@ -551,3 +551,70 @@ footer {
     margin: 0;
   }
 }
+
+.hamburger {
+  width: 1.5rem;
+  position: fixed;
+  top: 50px;
+  right: 60px;
+  cursor: pointer;
+  z-index: 100;
+}
+
+body header #desktop-nav nav {
+  position: fixed;
+  top: 3rem;
+  left: 1rem;
+}
+
+.nav-links {
+  padding-left: 0;
+}
+
+.nav-menu {
+  background-color: #fff;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  right: -100%;
+  display: flex;
+  justify-content: flex-start;
+  overflow: hidden;
+  transition: width 0.5s, height 0.5s, border-radius 0.5s;
+}
+
+.nav-menu.active {
+  right: 0;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+  z-index: 99999;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(6px) rotateZ(45deg);
+  border-bottom: 0.125rem solid #67798e;
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-6px) rotateZ(-45deg);
+  border-bottom: 0.125rem solid #67798e;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 3rem;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly make sure to pay attention to small details such as spacing and alignments of elements as these are the traits of a great front-end developer 
- [ ] As shown in the Figma design below the texts should be aligned to the left & the spacing between the texts is a bit too much so please try to reduce it. ✔
- Figma design:
  ![image](https://github.com/majdalkilany/Portfolio-/assets/104968236/9acd9513-86f1-4b0a-a45c-de867f5cb01f)
- Your design:
  ![Inkedfirefox_zb6b7Q0c9D](https://github.com/majdalkilany/Portfolio-/assets/104968236/4e0d5bfe-498a-4cab-b0b4-1d36e54ef75b)